client/JSDebugger/configurationProvider.ts

```diff
@@ -139,13 +141,18 @@ export class MLConfigurationProvider implements vscode.DebugConfigurationProvide
                     })
                 }
             }
-            const item = await vscode.window.showQuickPick(items, { placeHolder: 'Select the request to attach to' })
-            if (!item) {
-                return vscode.window.showErrorMessage('Request not selected').then(() => {
-                    return undefined	// abort
-                })
+            if (items.length > 0) {
+                const item = await vscode.window.showQuickPick(items, { placeHolder: 'Select the request to attach to' })
+                if (!item) {
+                    return vscode.window.showErrorMessage('Request not selected').then(() => {
+                        return undefined	// abort
+                    })
+                }
+                config.rid = item.label
+            } else {
+                vscode.window.showErrorMessage('No paused requests found on server')
+                return undefined
             }
```

Comment:
The above is the main purpose of this PR.
Everything in this file is formatting/ESLint-related.